data/transport missions.txt

```diff
@@ -1246,3 +1246,110 @@ mission "Terraforming 5"
 			`	"I just hope we haven't made things worse than they were," says Alaric. He hands you your payment of <payment>. "Anyway, thanks for your help, <first>. It sounds like there's nothing more to be done here for the time being."`
 		event "terraforming Rand"
+
+ "Building Skytower 1"
+	name "Building Skytower [1]"
+	description "Escort a convoy of freighters to <destination>, where they will pick up materials for an new skytower on Earth."
+	minor
+	source Earth
+	destination Alphard
+	to offer
+		random < 10
+                reputation republic: 1<
+                has "event: war ends: done"
+	passengers 1
+	
+	on offer
+		conversation
+			`As you are walking through the spaceport, you notice people crowding around windows. You walk to one of the less crowded windows and notice a tall black tower jutting out of the ground. This is no small feat of engineering, as even in it's incomplete state the building still dwarfes the others around it. It is quite apparently skytower`
+                        `As you are staring out the window, a man in a navy uniform walks up next to you. "Magnificent, isn't it?" He asks.`
+			choice
+				`	"Yes, indeed."`
+				`	"Is this a skytower?"`
+				
+			`	"It is," he says. "Now, I know what you're thinking. 'Wern't skytowers prohibited when the one in 2235 fell?' They were. But, with the ever growing population, our Earth is turning into an urban world. On top of that, the greenhouse gas emmisions from the billions of people that call this world home are destroying the atmosphere, and the oceans are rising steadily. We need to get lots of people off the planet, and fast too if we want to save our homeworld from becoming a ecological disaster!" He sounds very stressed, and you can tell this really stresses him out.`
+			choice
+				`	"This is scary stuff. Is there any way I can help?"`
+                                `       "Wow. What can we do?"`
+
+                       `        "There is somthing you can do. We need some cargo ships escorted to Alphard to pick up some super-strong graphine nanotubes that hopefully should help strengthen this." He gestures toward the incomplete tower. "I also need you to bring some scientists back as well. Are you up to the challange?"`
+                        choice
+                                `       "Yes I am!"`
+				`       "Sorry, but I have somthing urgent to do"` 
+                                decline	
+			`       "Thanks!" He says, "By the way, I'm admrial Michel Farford." "<first> <last>," you say. You shake hands, and you show Michel to his bunk."`
+	
+	on complete
+		payment 200000
+ 
+        npc
+		personality nemesis waiting
+		government Pirate
+		fleet "Large Northern Pirates"
+	npc
+		personality nemesis waiting
+		government Pirate
+		fleet "Small Northern Pirates"
+        npc
+		personality nemesis waiting
+		government Pirate
+		fleet "Large Northern Pirates"
+	npc
+		personality nemesis waiting
```

Comment:
Your Pirate fleets are in orbit around Earth at the moment. If you set them to `waiting` in systems between Earth and Alphard or around Alphard. They will actively seek out the player if they are not `staying`. Also as you get a navy license at the end of this I would recommend making it hard.
Last thing, you don't have any ships or fleets for the return trip. And Behemoths are quite tough so they may be quite easy to defend. If they are being use to build a large tower then you may want more of them.